modules/ROOT/pages/import-api-specification-exchange.adoc

```diff
@@ -55,8 +55,61 @@ Manage the API specifications linked to Exchange in your project from the API sp
++
+The *Join Beta* button is available only to your organization administrator.
+.. Reviewing the terms and conditions of the beta program
+.. Clicking *Get Access*
++
+When the AsyncAPI feature is available for your organization, the Anypoint Code Builder landing page provides the following notification:
++
+*AsyncAPI Implementation Beta for Anypoint Code Builder and Studio is enabled for your organization.*
+
+[NOTE]
+You must have access to a supported AsyncAPI specification hosted on Anypoint Exchange. If your AsyncAPI specification is not published to Exchange, see xref:design-center::design-publish.adoc[]. To create an AsyncAPI specification before publishing to Exchange, see xref:design-center::design-async-api.adoc[] in the Design Center documentation.
+
+=== Scaffolding Fundamentals for AsyncAPI Implementations
+
+When scaffolding an AsyncAPI specification into a Mule project (implementation or integration), Studio:
+
+* Introspects the AsyncAPI specification
+* Creates a new Mule project with a separate flow for each `publish` operation in the specification
+* Produces the configuration properties `file dev-properties.properties` file in the `src/main/resources` directory of your Mule project
+* Makes the APIkit for AsyncAPI module available for use in the project
++
+The module provides the following operations:
+
+** *Publish* (`<apikit-asyncapi:publish/>`): AsyncAPI `subscribe` operations in the specification are available for configuration as AsyncAPI *Publish* operations.
+** *Message Listener* (`<apikit-asyncapi:message-listener/>`): AsyncAPI `publish` operations in the specification are configurable *Message Listener* (not *Publish*) operations.
++
+The scaffolder transforms each `publish` operation into a *Message Listener* operation.
+
+* Generates a `global-configs.xml` file with connection configurations for *Message Listener* and *Publish* operations in the AsyncAPI Module and for any connectors that module depends on.
+
+Errors can occur during the scaffolding process:
+
+* If no channels are defined in the specification
+* If unsupported message broker protocols are used
+
+For supported protocols, see xref:apikit::apikit-asyncapi-module-reference.adoc#supported-message-brokers[Supported Message Brokers] in APIkit for AsyncAPI Module Reference.
+
+[NOTE]
+Studio doesn't support the scaffolding of AsyncAPI, OAS, or JSON schema fragment files that are referenced from API specs. The scaffolder doesn't add these fragments as project dependencies when you import the specification from Exchange. However, you can scaffold and reference fragments that are specified inline, within the spec. RAML fragments imported from Exchange are not affected by this limitation.
+
 == See Also
 
 * xref:api-development-studio.adoc[]
 * xref:api-sync.adoc[]
 * xref:set-credentials-in-studio-to.adoc[]
+* https://trailhead.salesforce.com/trailblazer-community/topics/mulesofteda?sort=LAST_MODIFIED_DATE_DESC[Provide Feedback about AsyncAPI feature]
```

Comment:
We usually don't put instructions in the See Also section. It might be more typical to add it to the main section.

Reply:
OK! Thanks!